docs/.c4/views/functional-layers/layers.views.c4

```diff
@@ -0,0 +1,99 @@
+views "uds-core-functional-layers" {
+  // Base Layer - Istio, UDS Operator, Policy Engine
+  view base-layer of cloud.cluster {
+    title "UDS Core Base Layer"
+
+    include
+      cloud,
+      cluster,
+      uds-core,
+      istio-system.*,
+      istio-tenant-gateway.*,
+      istio-admin-gateway.*,
+      pepr-system.*,
+      zarf.*,
+      kubernetes-control-plane.*
+  }
+
+  // Identity and Authorization Layer - Keycloak, AuthService
+  view identity-authorization-layer of cloud.cluster {
+    title "UDS Core Identity & Authorization Layer"
+
+    include
+      cloud,
+      cluster,
+      admin,
+      enduser,
+      uds-core,
+      uds-core.keycloak.*,
+      uds-core.authservice.*,
+      uds-core.istio-tenant-gateway.*,
+      uds-core.istio-admin-gateway.*,
+      cloud.cluster.external-services.ext-keycloak-db
+  }
+
+  // Metrics Server Layer (if applicable)
+  view metrics-server-layer of cloud.cluster {
+    title "UDS Core Metrics Server Layer"
+
+    include
+      cloud,
+      cluster,
+      uds-core,
+      kubernetes-control-plane.*
```

Comment:
metrics-server is/should be an actual separate namespace - as-is this diagram doesn't actually include metrics-server 👀